.github/workflows/ios.yml

```diff
@@ -0,0 +1,37 @@
+name: iOS CI
+
+on:
+  push:
+    branches: [ master]
```

Comment:
We don't have too many branches in the main repo- it might be nice to just test everything, that way we don't have to reconfigure things if we start working on a long running feature branch.